armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -107,6 +107,7 @@ adduser --system --group --disabled-login --no-create-home              electrs
 adduser --system --group --disabled-login --home /var/run/avahi-daemon  avahi
 adduser --system --group --disabled-login --no-create-home              prometheus
 adduser --system --group --disabled-login --no-create-home              node_exporter
+adduser --disabled-password --gecos "" hdmi
```

Comment:
Should adding the `hdmi` user, installing `chromium` etc be conditional, and disabled by default, maybe? That way, we'd avoid making the image / attack surface larger for users that don't need this feature.

Reply:
Adding this functionality should be done when building the image, and I think a flag to enable / disable makes sense (just not a high priority).

I don't think the attack surface is higher if the auto-login for the "hdmi" user (that has no password) is disabled and the x-server is not running, though. So it's more a question of ressource use in my opinion.